pkg/p2p/server.go

```diff
@@ -408,7 +422,7 @@ func (m *MessageServer) AddHandler(
 				zap.Int64("lastAck", lastAck))
 			return nil
 		}
-
+		log.Info("sequence", zap.Int64("seq", entry.Sequence))
```

Comment:
Will this log be useful if the log level is not `debug`.

Reply:
removed